charts/trusted-attestation-controller/values.yaml

```diff
@@ -0,0 +1,98 @@
+nameSpace: attestation-controller
+
+secrets:
+  caCrt:
+  clientCrt:
+  clientKey:
+
+  name: tac-null-secrets # Change based on plugin usage
+
+deployment:
+  replicaCount: 1
+
+  imageTag: latest
+
+  pullPolicy: Always
+
+  pluginArgs:
+    - --plugin-name=null
+
+  port: 8081
+
+  livenessProbe:
+    initialDelaySeconds: 15
+    periodSeconds: 20
+  readinessProbe:
+    initialDelaySeconds: 5
+    periodSeconds: 10
+  
+  limits:
+    cpu: 200m
+    memory: 100Mi
+
+  requests:
+    cpu: 100m
+    memory: 20Mi
+  
+  volumeMounts: # Change based on plugin usage
+  - mountPath: /null
+    name: plugin-socket-dir
+  - mountPath: /registration
+    name: registry-socket-dir
+
+plugin: # Change based on plugin usage, defaults to the null-plugin
+  name: null-plugin
+
+  args:
+    - --plugin-name=null
+    - --plugin-socket-path=/null/null.sock
+
+  command:
+    - /null-plugin 
+
+  limits:
+    cpu: 500m
+    memory: 100Mi
+  requests:
+    cpu: 100m
+    memory: 20Mi
+  
+  volumes:
```

Comment:
Do we need to expose volumes and volumeMounts? Do they change from plugin-to-plugin?

Reply:
Atleast the mount path and secret name changes based on plugin usage. Also the socket dir needs to be exposed for the plugin to registry with the manager.

The secret volumes are not needed in the null plugin case so ill comment those out for now?